### DISCUSSION TITLE
"Entry URI Format" auto-population causes UX problems

### DISCUSSION THREAD

**Discussion by zenbug** [craftcms/cms#8884] [Ideas]
### Description

Some Sections' entries shouldn't have their own URLs, such as a section that's being used as a category. Craft lets me indicate this by leaving the "Entry URI Format" field blank when creating the Section. Indicating that certain Sections shouldn't have URLs is important for some plugins, such as SEOmatic, which will automatically prompt you to configure SEO settings for any Section that *does* have URLs. But the instructions that explain how to disable URLs are somewhat hidden.

One way to find these instructions is to click on the "i" icon next to the "Entry URI Format" field. The problem is that this field is auto-populated as soon as I type a name for the Section, so I am unlikely to click on the "i" icon. The auto-population makes me think that Craft needs this field to be populated, like the Handle field, so I might just let it auto-populate.

I'd suggest having more prominent instructions explaining that I should leave the "Entry URI Format" field blank if I don't want that Section to have its own URLs. Maybe just have that text written above the field.

I also noticed that when adding a new Category, the "Entry URI Format" field contains the placeholder text "Leave blank if categories don't have URLs." For consistency, I should probably see the same placeholder when adding a new Section (though this still doesn't solve the problem, since that text will be replaced when I type a Section name).

**Comment by brandonkelly** [comment]
Will have to play around with how to improve this, but for now I went ahead and added that placeholder text, which should help a bit.

**Comment by zenbug** [comment]
Excellent. Thanks for the prompt action, Brandon.
